features/FxA-45-device-registration-api/README.md

```diff
@@ -0,0 +1,386 @@
+# Device registration API
+
+https://mozilla.aha.io/features/FXA-45
+
+* [Stories](#stories)
+* [Data](#data)
+* [API](#api)
+  * [POST /v1/account/create](#post-v1accountcreate)
+  * [POST /v1/account/login](#post-v1accountlogin)
+  * [POST /v1/account/device](#post-v1accountdevice)
+  * [GET /v1/account/devices](#get-v1accountdevices)
+  * [POST /v1/account/device/destroy](#post-v1accountdevicedestroy)
+* [Details](#details)
+  * [How are shared devices handled?](#how-are-shared-devices-handled)
+  * [Are shared devices bound to the account or the device?](#are-shared-devices-bound-to-the-account-or-the-device)
+  * [How are unrecognised devices handled?](#how-are-unrecognised-devices-handled)
+  * [What is a disconnected device?](#what-is-a-disconnected-device)
+
+## Stories
+
+* As an API consumer,
+  I want to create an account with the auth server
+  using new device details
+  and receive the newly-registered device id in the response.
+
+* As an API consumer,
+  I want to create an account with the auth server
+  using a registered device id.
```

Comment:
Oh yeah, I think I saw that when I looked at the schema but failed to think it through. Have removed this and all the shared device gubbins.